_shared/project/setup.cfg

```
@@ -20,25 +20,26 @@ packages = find:
 python_requires = >={{ python_versions()|oldest|pyformat(PyFormats.MAJOR_DOT_MINOR_FMT) }}
 install_requires =
     importlib_metadata;python_version<"3.8."
-    {%- if include_exists("setuptools/install_requires") %}
-      {{- include("setuptools/install_requires", indent=4) }}
-    {%- endif %}
+{% if include_exists("setuptools/install_requires") %}
+{{ include("setuptools/install_requires", indent=4) -}}
+{% endif %}
```

```suggestion
{% if include_exists("setuptools/install_requires") %}
    {{ include("setuptools/install_requires", indent=4) -}}
{% endif %}
```

Can we indent now we have whitespace stripping?

---

Since this is a single-line block I think we should be able to indent this one if we used a `{{-`, yes. I'm not sure which approach I want to go with for consistency: always indent single-line block bodies with a `{{-`? Or never indent them so that you don't need the `-`? Multi-line block bodies cannot be indented.

---

I've gone through and indented the expression with a `{{-` in all cases where we have a block whose body is just a single expression line (there's only a handful of opportunities to do this)